translation-files/German/bloodtears_bureau_l_german.yml

```diff
@@ -1,71 +1,67 @@
+flight_officers_training1: "Flugmissionen Maximum"
+flight_officers_training1_desc: "Indem wir ein Limit setzen, wie oft unsere Piloten an Missionen teilnehmen dürfen, können wir diese erfahrenen Piloten benutzen, Neue auszubilden, wodurch die durchschnittliche Gesamtleistung unserer Luftwaffe erhöht wird.\n\n"
+flight_officers_honor1: "Ehre oder Tod"
+flight_officers_honor1_desc: "Unsere Fliegerasse sind unsere modernen, legendären Helden; große Krieger, representant für den Willen und die Macht der Luftwaffe unserer Nation. Wenn die Piloten des Gegners ihre Namen hören, kauern sie in Angst.\n\n"
+flight_officers_honor2: "Ehre oder Tod"
+flight_officers_honor2_desc: "Unsere Piloten werden das letzte Opfer für den Ruhm unserer Nation erfüllen. Unsere Soldaten sollen dem Tod begegnen, aber unsere heroischen Piloten sollen ihn umarmen."
 
-flight_officers_honor2_special: "\n\n\n\n\n\n\n\nAllows Kamikaze Strikes air missions, suicide attacks against enemy ships."
+flight_officers_honor2_special: "\n\n\n\n\n\n\n\nErlaubt Kamikaze-Einsätze und Selbstmordattacken gegen feindliche Schiffe."
 
 # Press Tech
 
-tactics_TITLE_civilian: "CIVILIAN BUREAU"
-civilian_technology_desc: "Our civilian ministries and propaganda can help us to mold our civilian population into a helpful tool to wield against our enemies."
+tactics_TITLE_civilian: "Ziviles Amt"
+civilian_technology_desc: "Unsere zivilen Ministerien und Propaganda können uns helfen, unsere Zivilbevölkerung zu einem nützlichen Werkzeug zu machen, um gegen unsere Feinde zu handeln."
 
-press_tech1: "Press Integration"
-press_tech1_desc: "Our press can serve our state. If we let the public in on our public plans and put the right spin on it, we can direct the public support (and anger) where we need it; even if this means lying to them using alternative facts."
+press_tech1: "Presse-Integration"
+press_tech1_desc: "Unsere Presse kann unserem Staat dienen. Wenn wir die Öffentlichkeit in unsere öffentlichen Pläne einlassen und den richtigen Punkt darauf setzen, können wir die öffentliche Unterstützung (und Wut) einsetzen, wo wir sie brauchen, auch wenn es darum geht, sie mit alternativen Fakten zu belügen."
 
-press_censorship1: "Censorship"
-press_censorship1_desc: "Using censorship we can suppress delicate information and reduce the intel the enemy's resistance has access to and thus reduce their power and impact on our troops. Further such efforts and instilling the fear of spies in our population will make it harder for actual hostile agents to work as effectively."
-press_opposition_surpression: "Opposition Suppression"
-press_opposition_surpression_desc: "By censoring and isolating our political opponents, we can keep foreign powers and hostile ideologies influence and corrupt our nation."
```

Comment:
Oh, just noticed, you mixed up `press_media2` with the new `press_opposition_surpression`.
In the latest update I split those two. `press_media2` is now called `press_opposition_surpression`, and `press_media2` is something new (that still needs to be written)